projects/valtimo/plugin/src/lib/plugins/objecttypen-api/components/objecttypen-api-configuration/objecttypen-api-configuration.component.html

```diff
@@ -0,0 +1,72 @@
+<!--
```

Comment:
File should be formatted with prettier

Reply:
Weird if it is incorrect. I have enabled a on reformat code action and on save for prettier in intellij